docs/tooling/packaging.md

```diff
@@ -0,0 +1,51 @@
+# Packaging PaSh
+
+`scripts/package` packages PaSh into various formats. It is a suitable
+replacement for deployments depending on `install.sh`,
```

Comment:
`install.sh` is no longer supported for pash setup process

Reply:
What's canonical?

Reply:
`distro-deps.sh` with `setup-pash.sh` and `up.sh`

Reply:
Can macOS users run `up.sh` without issue, or is there an existing macOS deployment effort I can reference? System-level macOS deployment is not as straightforward as the other platforms.

Reply:
Currently, macOS is not supported (several things are broken). I don't think we should bother with packages for mac for the time being until the installation process for mac is stable but also all the tests past without fails.

Reply:
Makes sense, thanks.